packages/vats/test/bootstrapTests/test-vaults-upgrade.js

```diff
@@ -67,7 +67,7 @@ const makeDefaultTestContext = async t => {
 test.before(async t => {
   t.context = await makeDefaultTestContext(t);
 });
-test.after.always(t => t.context.shutdown());
+test.after.always(t => t.context.shutdown?.());
```

Comment:
this seems like a bad idea to me. Every bootstrap test needs to shutdown the host storage.

Reply:
ok. gone.